File: Dart/Dart OOPs/ClassObject.dart
// Dart - Class & Object 
void main(List<String> args) {
  // class Object 
  Test_Class tc = new Test_Class();
  var tc1 = Test_Class();
  
  tc.methode();
  tc1.methode();
}
class Test_Class{
  // class data
  int n = 100;
  String s = 'string';

  // class methode 
  void methode(){
    print('class Test methode ');
  }
}

File: CPP/CPP OOP/ClassObject.cpp
// cpp - Class & Object
#include <iostream>
// class definition
class Test_Class
{
    // Data & Function as-per access modifier
private:
    int private_num = 100;
    void set(int n1, int n2)
    {
        this->private_num = n1;
        this->public_num = n2;
    }

protected:
public:
    int public_num = 1;

    void display()
    {
        std::cout << "public num = " << public_num << "\tPrivate num = " << private_num << std::endl;
    }
};
int main()
{
    // Object of class
    Test_Class tc;
    tc.display();

    // pointer based object
    Test_Class *tc1 = new Test_Class;
    tc1->display();
    return 0;
}